### ISSUE TITLE
Add a pop up instruction on how to play the game

### ISSUE THREAD

**Issue by hassana123** [yelynn1/tictactoe#72]
when a user clicks on start game a pop up instruction on how to play the game should be displayed

**Comment by Shivangbhatnagar237** [comment]
I think this issue has been already resolved as the Instructions on how to play the game are displayed even before the user clicks on 'Start Game'

**Comment by hassana123** [comment]
Yeah becaus I added it. And if you read the contribution guidelines you will know that's one is required to raise an issue before contributing
